.gemini.yml

```diff
@@ -0,0 +1,25 @@
+rootUrl: http://127.0.0.1:3000/test/visual
+gridUrl: http://127.0.0.1:4444/wd/hub
+screenshotsDir: ./test/visual/screens
+
+system:
+  plugins:
+    express:
+      port: 3000
+    sauce: true
```

Comment:
Ok maybe I oversimplified a bit. So in order to run the test locally, you'll need to

1. Comment out the plugin
2. Start up Selenium locally
3. Comment out everything not locally supported from the browser configs
4. Run "gemini test test/visual"